Tests/AppcuesKitTests/Experiences/ExperienceStateMachineTests.swift

```diff
@@ -176,6 +181,34 @@ class ExperienceStateMachineTests: XCTestCase {
         XCTAssertEqual(stateMachine.state, .idling)
     }
 
+    func test_stateIsRenderingStep_onViewControllerDismissed_doesNotMarkComplete() throws {
+        // the @appcues/skippable trait would do this
+
+        // Arrange
+        let dismissExpectation = expectation(description: "Experience dismissed")
+        let experience = ExperienceData.singleStepMock
+        let package: ExperiencePackage = experience.package(dismissExpectation: dismissExpectation)
+
+        let initialState: State = .renderingStep(experience, Experience.StepIndex(group: 0, item: 0), package, isFirst: true)
```

Comment:
Should this be tested from the last step in the Mock experience (`group: 0, item: 0`) to confirm that skipping the last step gets us the `experience_dismissed` analytic?

Reply:
do you mean `mock` instead of `singleStepMock` and use (`group: 1, iem: 0`) ?